src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -137,6 +150,20 @@ export default {
 		hasTrustee() {
 			return false;
 		},
+		loan() {
+			return this.apollo.readFragment({
+				id: this.loanId,
+				fragment: detailedLoanCardFragment,
+			});
+		},
+		retinaImageUrl() {
+			// eslint-disable-next-line quotes
+			return this.loan.image.retina.replace(`/w960h600/`, `/w960h720/`);
+		},
+		standardImageUrl() {
+			// eslint-disable-next-line quotes
+			return this.loan.image.default.replace(`/w480h300/`, `/w480h360/`);
```

Comment:
This can fail if `this.loan` is ever null or undefined

Reply:
Oh yes, nice eye. I'm putting in a couple of guards before merging :+1: